spyder_terminal/server/logic/term_manager.py

```diff
@@ -52,29 +55,32 @@ def __init__(self):
         """Main terminal handler constructor."""
         self.os = os.name
         if self.os == WINDOWS:
-            self.cmd = 'cmd'
-            self.pty_fork = lambda x: pspawn.PopenSpawn(x, encoding="utf-8")
+            self.cmd = r'C:\windows\system32\cmd.exe'
```

Comment:
I think this should be defined via plugin configuration settings [spyder#4306](https://github.com/spyder-ide/spyder/issues/4306)